README.md

```diff
@@ -186,7 +186,10 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Inference](https://github.com/roboflow/inference) - A fast, production-ready inference server for computer vision supporting deployment of many popular model architectures and fine-tuned models. With Inference, you can deploy models such as YOLOv5, YOLOv8, CLIP, SAM, and CogVLM on your own hardware using Docker.
 * [Jina](https://github.com/jina-ai/jina)  ![](https://img.shields.io/github/stars/jina-ai/jina.svg?style=social) - Cloud native search framework that   supports to use deep learning/state of the art AI models for search.
 * [KServe](https://github.com/kserve/kserve) ![](https://img.shields.io/github/stars/kserve/kserve.svg?style=social) - Serverless framework to deploy and monitor machine learning models in Kubernetes - [(Video)](https://www.youtube.com/watch?v=hGIvlFADMhU).
+* [Langfuse](https://github.com/langfuse/langfuse) ![](https://img.shields.io/github/stars/langfuse/langfuse.svg?style=social) - Langfuse is an open source observability & analytics solution for LLM-based applications.
+* [LangKit](https://github.com/whylabs/langkit) ![](https://img.shields.io/github/stars/whylabs/langkit.svg?style=social) - LangKit is an open-source text metrics toolkit for monitoring language models.
```

Comment:
Seems thin wrapper on closed source tooling (similar to W&B) as opposed to OSS so more relevant to Commercial section

Reply:
Thanks for the quick reply.
I checked the readme but still failed to find any strong closed-source project signal.
![image](https://github.com/EthicalML/awesome-production-machine-learning/assets/8592144/1df5386d-3f9c-46b8-a141-2d5b876363c9)

Reply:
OK - let me have a deeper look

Reply:
Here: 
<img width="858" alt="image" src="https://github.com/EthicalML/awesome-production-machine-learning/assets/1447507/6bc75c7d-f57d-42b6-a327-5c51ca623e3d">

Reply:
I check https://github.com/whylabs/whylogs but find it is still an open-sourced project.

Reply:
Ok good to know, it may have slipped, I do know that WhyLogs is indeed a closed source platform https://whylabs.ai/ - given this then we should review these entries, most likely it would be just removing these two (we can consider packages that offer full functionality as standalone, an example is MLFlow, which indeed has a managed service provided by Databricks but can still provide full functionality by itself with the OSS package without subscription)